src/test/java/edu/harvard/iq/dataverse/api/FilesIT.java

```diff
@@ -2483,4 +2485,122 @@ public void testCollectionStorageQuotas() {
         
         UtilIT.deleteSetting(SettingsServiceBean.Key.UseStorageQuotas);
     }
+
+    @Test
+    public void testFileCitation() throws IOException {
```

Comment:
To be consistent with the API method name:

```suggestion
    public void testGetFileCitationByVersion() throws IOException {
```

Reply:
Yes, good idea.

Reply:
Done as part of this change: e9ab40bdaa simplify and rename tests